vuepress/docs/next/tutorials/solution/entando-hub.md

```diff
@@ -65,24 +64,79 @@ ent ecr deploy --repo=docker://registry.hub.docker.com/entando/entando-hub-conte
+
+### Create New Entries/Bundle Groups
+Click the `Add +` button at the top of the Hub UI home page to create a new bundle group. In the pop-up window, enter the details for the entry.
+
+![hub-add.png](./hub-images/hub-add.png)
+
+
+1. Upload a file of the thumbnail for the bundle group. 
+2. Add one or more bundles for the entry using the `Add +` button next to the `Add Bundle URI` field.  
+3. Check `Display Contact Us button` and enter the URL under `Contact URL` to gather more information from the viewer/visitor and manage access to the entry. Typically, the contact URL points to a web form on the owner's web site with a request for access to the entry.
+
+### Create a Private Catalog
+A private catalog can be configured in the Hub UI when creating a new organization. There can be many organizations in a single Hub instance, but each organization is allowed one private catalog. Only the Hub admin can create an organization and provide a private catalog for it. 
+1. Go to Organization Management from the top menu
+2. Click `Add Organization +`, enter the relevant information in the pop-up window, and click `Save`
+3. The new organization will appear in the current list. Click on the kebab menu to the right of the organization and select `Create Private Catalog`. 
+A key icon will appear next to the private catalog. To go directly to this catalog, there is a link under the same kebab menu. 
+
+### Generate an API Key
+API access to private catalogs requires the use of an API key instead of user credentials. When connecting a registry from the App Builder, the API key is required to configure a private catalog.
+1. API Keys are attached to a specific user account so login as a user assigned to the organization of the private catalog.
+2. From the Hub UI homepage, click on the gear icon right of the `Add +` button and select `API Key Management`.
+3. Click `Generate API Key`, enter a name, and confirm with the blue generate button. Save the key for future reference.
 
-### Concepts
+### Add a Catalog as a Registry in your App Builder
+Any enterprise Hub instance can be accessed from the Entando App Builder of another Entando Application. Configure the App Builder to access the desired catalog via the endpoint `BASEURL/entando-hub-api/appbuilder/api`, where the BASEURL is the address for the Entando Application. 
 
+1. Go to the Hub from the left navigation bar in the App Builder and click `Select Registry` 
+2. Choose `New Registry` from the drop-down menu
+3. Enter a name and the access URL for the catalog.
+     * Private Catalog
+        1. The access URL for a private catalog follows the pattern noted above, but with an added ID number from the catalog's HTTP address. Go to the published catalog page from the App Builder and find the address in the browser. The number after `/catalog/` is `YOUR-CATALOG-ID#`.  
+        2. The URL to access the catalog is `BASEURL/entando-hub-api/appbuilder/api/?catalogId=YOUR-CATALOG-ID#`  
```

Comment:
It looks like `/entando-hub-api/` has to change to `/entando-hub-application-152edaba/entando-hub-catalog-ms/`
CC: @firegloves @jyunmitch 

I wish we'd kept the ingress path in place so we could control the path in the entando.json but I think this is okay right now.